typing/typetexp.ml

```diff
@@ -336,7 +336,8 @@ end = struct
                   raise (Error(loc, env, Type_mismatch err))
               | _ -> Btype.backtrack snap; false
           then try
-            r := (loc, v, lookup_global_type_variable name) :: !r
+            r := (loc, v, lookup_global_type_variable name) :: !r;
```

Comment:
When there is only one source of exceptions, and many expressions  I personally tend to prefer  the form
```
match lookup_global_type_variable name with
| global -> r := ...; unused :=
| exception Not_found -> ...
```
but this is personal taste.